testing/file_test/autoupdate.cpp

```diff
@@ -43,6 +43,7 @@ FileTestAutoupdater::FileAndLineNumber::FileAndLineNumber(
       line_number(ParseLineNumber(line_number)) {}
 
 auto FileTestAutoupdater::CheckLine::RemapLineNumbers(
```

Comment:
Can you add a unit test of the changes here, to testing/file_test/testdata?

Reply:
Done.